docker-compose.yml

```diff
@@ -10,7 +10,7 @@ services:
         - POSTGRES_PASSWORD=balancer
         - POSTGRES_DB=balancer_dev
       ports:
-      - "5432:5432"
+      - "5433:5432"
```

Comment:
@taichan03  Did we have to use a different host port because `5432`  was already being used by a service?